ssm/crypto.py

```diff
@@ -180,6 +180,9 @@ def verify(signed_text, capath, check_crl):
         log.debug(error)
     else:
         log.warn(error)
+        raise CryptoException(
+            "Failed to verify the signed message."
```

Comment:
```suggestion
            "Possible tampering. See OpenSSL error: %s" % error
```
This message should probably contain the full error as the error handling when this is called already says pretty much this:

https://github.com/apel/ssm/blob/a75ded59967e24f694429760bdce960467dedaa3/ssm/ssm2.py#L329